src/client/app/access-checker/admin-cache-entries.component.ts

```diff
@@ -37,8 +38,8 @@ export class AdminCacheEntriesComponent {
 		this.alertService.clearAllAlerts();
 
 		this.pagingOpts = new PagingOptions();
-		this.pagingOpts.sortField = this.sortOpts.key.sortField;
-		this.pagingOpts.sortDir = this.sortOpts.key.sortDir;
+		this.pagingOpts.sortField = this.sortOpts['key'].sortField;
```

Comment:
Weird. That change is not in the current version of the branch (Typescript 2.2 allows dot notation for string indexers, while Typescript 2.1) does not. Not sue why the code review showed ['key'] for you.